uui/components/fileUpload/DropSpot.tsx

```diff
@@ -9,10 +9,25 @@ import { i18n } from '../../i18n';
 import css from './DropSpot.module.scss';
 
 export interface DropSpotProps extends IHasRawProps<React.HTMLAttributes<HTMLDivElement>> {
+    /*
+    * CSS class(es) to put on component's root. See {@link https://github.com/JedWatson/classnames#usage} for details
+    */
     cx?: string;
+    /*
+    * Defines the infoText.
+    */
     infoText?: string;
+    /*
+    * Callback to processing upload files
+    */
     onUploadFiles(files: File[]): any;
```

Comment:
Rework by guidelines - https://github.com/epam/UUI/wiki/Documentation-Guidelines#api-design-and-documentation-guidelines

Reply:
fixed